cmd/server/assets/admin/new.html

```diff
@@ -0,0 +1,103 @@
+{{define "admin/newrealm"}}
+
+{{$realm := .realm}}
+
+<!doctype html>
+<html lang="en">
+<head>
+  {{template "head" .}}
+</head>
+
+<body>
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>New Realm</h1>
+    <p>
+      Use the form below to create a new realm.
+    </p>
+
+    <div class="card mb-3">
+      <div class="card-header">Details</div>
+      <div class="card-body">
+        <form method="POST" action="/admin/realms/create">
+          {{ .csrfField }}
+
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Name:</label>
+            <div class="col-sm-9">
+              <input type="text" id="name" name="name" class="form-control{{if $realm.ErrorsFor "name"}} is-invalid{{end}}" value="{{$realm.Name}}">
+              {{if $realm.ErrorsFor "name"}}
+              <div class="invalid-feedback">
+                {{joinStrings ($realm.ErrorsFor "name") ", "}}
+              </div>
+              {{end}}
```

Comment:
Should we add the help text beneath?

Reply:
the isn't help text for name, but I added the region code one.